src/views/pengajuanLogistik/detail.vue

```diff
@@ -384,6 +426,12 @@ export default {
     this.letterFileType = temp[temp.length - 1]
     this.isVerified = this.detailLogisticRequest.applicant.verification_status === 'Terverifikasi'
     this.isRejected = this.detailLogisticRequest.applicant.verification_status === 'Pengajuan Ditolak'
+    this.isApproved = this.detailLogisticRequest.applicant.approval_status === 'Telah Disetujui'
+    if (this.detailLogisticRequest.applicant.stock_checking_status === 'checked') {
+      this.isStock = true
+    } else {
+      this.isStock = false
+    }
```

Comment:
Percabangan ini mungkin bisa disingkat menjadi
```suggestion
    this.isStock = this.detailLogisticRequest.applicant.stock_checking_status === 'checked'
```